docs/semgrep-app/findings.md

```diff
@@ -167,13 +167,36 @@ To **remove a rule** from the Rule Board:
 4. Click **Save**.
 5. Manually initiate a Semgrep scan from your CI job or SCM tool to immediately remove findings generated by the removed rule.
 
-To **view and add comments**:
+To **view and add comments** to history of a finding:
 
 1. Click the <i className="fa-regular fa-arrow-up-right-from-square inline_svg"></i> **Finding details** icon next to the finding’s status.
 2. Find the comments in the **History** section.
     ![Semgrep App finding details page](/img/app-finding-details.png)<br />
     *Figure 4.* Findings details page.
 
+### Ignoring findings through comments
+
+Triage your Semgrep App findings that are displayed as comments in GitHub PRs by replying with another comment.
```

Comment:
This feature should only work for private GitHub PRs. I'm not sure if we specify this elsewhere but maybe we could specify it here again?

Reply:
True! Thank you for the reminder @pangngan. :)

```suggestion
Triage your Semgrep App findings displayed as comments in GitHub PRs by replying with another comment.

:::info Prerequisite
Private GitHub repository added as a project in Semgrep App. This feature is not enabled for public GitHub repositories. For information on adding a repository through Semgrep App, see [Getting started with Semgrep App](/semgrep-app/getting-started-with-semgrep-app/).
:::
```